packages/node/src/integrations/utils/http.ts

```diff
@@ -101,7 +103,8 @@ export function urlToOptions(url: URL): RequestOptions {
     options.port = Number(url.port);
   }
   if (url.username || url.password) {
-    options.auth = `${url.username}:${url.password}`;
+    // always filter authority, see https://develop.sentry.dev/sdk/data-handling/#structuring-data
+    options.auth = '[Filtered]:[Filtered]';
```

Comment:
I expect `[Filtered]` to be refactored out to a shared const as more filtering is introduced client side, likely via #5347. If I should do that here, please let me know where you'd recommend the constant to live. From what I can tell, this is the first/only client-side filtering in sentry-js